exporter/sentryexporter/sentry_exporter.go

```diff
@@ -391,6 +398,14 @@ func transactionFromSpan(span *sentry.Span) *sentry.Event {
 	return transaction
 }
 
+func generateEventID() sentry.EventID {
+	id, _ := uuid.New().MarshalBinary()
+	out := make([]byte, 32, 32)
+	hex.Encode(out, id)
+
+	return sentry.EventID(out)
+}
+
```

Comment:
Prefer approach to https://github.com/getsentry/sentry-go/blob/8a528f85d1952e6690c627caec2efe7f010a6964/util.go#L15 so we don't have to bring in the uuid dep.